File: Examples/Node.js/SDK/manipulate/ConvertTiffImagetoFaxFormat.js
var fs = require('fs'),
    utils = require('../utils.js'),
    image = utils.imagingApi;
    storage = utils.storageApi;

var inputName = "sample1.tiff";
var inputPath = utils.getPath(__filename, inputName);

var outputName = "nodejsSDK.output.tiff";
var outputPath = utils.getPath(__filename, outputName);

var Storage = null;
var folder = null;

storage.PutCreate(inputName, null, Storage, inputPath, function (responseMessage) {
    image.GetTiffToFax(inputName, Storage, folder, null, function(responseMessage) {
        storage.GetDownload(inputName, null, null, function (responseMessage) {
            var writeStream = fs.createWriteStream(outputPath);
            writeStream.write(responseMessage.body);
        });
    });
});

File: Examples/Python/SDK/manipulate/ConvertTiffImagetoFaxFormat.py
import common

input_file = "sample1.tiff"
input_path = common.get_path(__file__, input_file)

output_file = "output.tiff"
output_path = common.get_path(__file__, output_file)

common.StorageApi.PutCreate(input_file, input_path)
# invoke Aspose.Imaging Cloud SDK API to perform format change operation without using cloud storage
response = common.imagingApi.GetTiffToFax(input_file)

if response.Status == 'OK':
    # download image from API response
    with open(output_path, 'wb') as f:
        for chunk in response.InputStream:
            f.write(chunk)
